packages/sw-background-sync-queue/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sw-background-sync-queue",
+  "version": "0.0.1",
+  "description": "A service worker implementation of the behavior defined in a page's App Cache manifest.",
+  "keywords": [
+    "service worker",
+    "sw",
+    "offline",
+    "queue"
+  ],
+  "author": {
+    "name": "Prateek Bhatnagar",
+    "email": "prateekbh@google.com"
+  },
+  "license": "Apache-2.0",
+  "repository": "googlechrome/sw-helpers",
+  "bugs": "https://github.com/googlechrome/sw-helpers/issues",
+  "homepage": "https://github.com/GoogleChrome/sw-helpers/tree/master/projects/sw-background-sync-queue",
+  "main": "background-sync-queue.js",
+  "jsnext:main": "background-sync-queue.mjs"
```

Comment:
👍 for the `module` property. (Side: we might want to document some of these practices for future subprojects somewhere)